src/tools/sample/sample.c

```diff
@@ -994,6 +1003,90 @@ RunClient(
     }
 }
 
+
+//
+// Runs the multi client side of the protocol.
+//
+void
+RunMultiClient(
+    _In_ int argc,
+    _In_reads_(argc) _Null_terminated_ char* argv[]
+    )
+{
+    //
+    // Load the client configuration based on the "unsecure" command line option.
+    //
+    if (!ClientLoadConfiguration(GetFlag(argc, argv, "unsecure"))) {
+        return;
+    }
+    QUIC_STATUS Status;
```

Comment:
nit: I recommend default initializing the variables - for this one, it would be pretty easy to reach the cleanup without having it set to a value.

Reply:
Usually, the static analysis catches this if it's an actual issue. But yeah, in the success case, I see this makes it through the whole function without assigning to `Status`. Fixed.